src/index.html

```diff
@@ -7,6 +7,56 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section>
+      <div class="calendar">
+        <div class="start-day">
+          <div class="start-day__day start-day__day--mon"></div>
+          <div class="start-day__day start-day__day--tue"></div>
+          <div class="start-day__day start-day__day--wed"></div>
+          <div class="start-day__day start-day__day--thu"></div>
+          <div class="start-day__day start-day__day--fri"></div>
+          <div class="start-day__day start-day__day--sat"></div>
+          <div class="start-day__day start-day__day--sun"></div>
+        </div>
```

Comment:
you don't need these `divs`. it should look like that
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/102411130/220043514-abc80cd6-8928-4f99-8226-2aa826b4d96a.png">

```suggestion
      <div class="calendar calendar--start-day_mon calendar--month-length_31">
```